packages/cli-hydrogen/package.json

```diff
@@ -48,7 +48,8 @@
     "graphql-request": "^4.3.0",
     "prettier": "^2.6.1",
     "typescript": "^4.7.4",
-    "vite": "^2.9.9"
+    "vite": "^2.9.9",
+    "enquirer": "^2.3.6"
```

Comment:
I'd advise not to introduce `enquirer`. It was our default in the past but [we replaced it with inquirer](https://github.com/Shopify/shopify-cli-planning/issues/298) because of issues in Linux environments.